[17 01 2017 12:37:47.706] [DEBUG] - Mongoose is now disconnected..!
[17 01 2017 12:37:47.709] [ERROR] - Error in Mongoose connection:  { MongoError: failed to connect to server [127.0.0.1:27017] on first connect
    at Pool.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\topologies\server.js:326:35)
    at emitOne (events.js:96:13)
    at Pool.emit (events.js:188:7)
    at Connection.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\pool.js:272:12)
    at Connection.g (events.js:291:16)
    at emitTwo (events.js:106:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (E:\simanta\Immersive\react-serverapp-boilerplate\node_modules\mongodb-core\lib\connection\connection.js:175:49)
    at Socket.g (events.js:291:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1278:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
  name: 'MongoError',
  message: 'failed to connect to server [127.0.0.1:27017] on first connect' }
[17 01 2017 12:42:17.706] [INFO] - Mongoose disconnected on process termination
